src/common/NetworkPrivate.cpp

```diff
@@ -181,13 +186,38 @@ void loadUncached(const std::shared_ptr<NetworkData> &data)
                     QNetworkReply::NetworkError::OperationCanceledError)
                 {
                     // Operation cancelled, most likely timed out
+                    qCDebug(chatterinoHttp)
+                        << QString("%1 %2 [cancelled]")
+                               .arg(networkRequestTypes.at(
+                                   int(data->requestType_)))
+                               .arg(data->request_.url().toString());
                     return;
                 }
 
                 if (data->onError_)
                 {
                     auto status = reply->attribute(
                         QNetworkRequest::HttpStatusCodeAttribute);
+                    if (data->requestType_ == NetworkRequestType::Get)
+                    {
+                        qCDebug(chatterinoHttp)
+                            << QString(
+                                   "%1 %3 %2")  // reversed to not break URLs
+                                   .arg(networkRequestTypes.at(
+                                       int(data->requestType_)))
+                                   .arg(status.toInt())
+                                   .arg(data->request_.url().toString());
```

Comment:
Here, and in all other places with `QString::arg` methods we should use multi-arg instead, see https://github.com/KDE/clazy/blob/master/docs/checks/README-qstring-arg.

(I'll make a PR fixing all those other places with this in codebase later.

Reply:
1c9e6f2 (will resolve when and if it compiles)